src/runtime/hexagon_host.cpp

```diff
@@ -755,6 +755,13 @@ WEAK uint64_t halide_hexagon_get_device_size(void *user_context, struct halide_b
     return handle->size;
 }
 
+WEAK void *halide_hexagon_get_module_state(void **host) {
+    if (host == nullptr) {
+        return nullptr;
```

Comment:
Is `nullptr` a value that the caller can deal with intelligently? (i.e., is this an error condition?)

Reply:
I'm not quite sure. This check doesn't exist in the current implementation, so perhaps I should remove it. I wanted to avoid null pointer dereferencing.

Reply:
Always a good idea. Might be worth throwing an `abort()` or similar in the null path to see?

Reply:
Sure, just tried it out in 2df1b2cdc